<launch>
  <arg name="device" default=""/>
  <arg name="bitrate" default="128"/>
  <arg name="channels" default="1"/>
  <arg name="sample_rate" default="16000"/>
  <arg name="dst" default="/tmp/output.mp3"/>
  <arg name="format" default="mp3"/>
  <arg name="sample_format" default="S16LE"/>
  <arg name="ns" default="audio"/>

  <group>
    <push-ros-namespace namespace="$(var ns)"/>
    <node name="audio_capture" pkg="audio_capture" exec="audio_capture_node" output="screen">
      <param name="bitrate" value="$(var bitrate)"/>
      <param name="device" value="$(var device)"/>
      <param name="channels" value="$(var channels)"/>
      <param name="sample_rate" value="$(var sample_rate)"/>
      <param name="dst" value="$(var dst)"/>
      <param name="sample_format" value="$(var sample_format)"/>
      <param name="format" value="$(var format)"/>
    </node>
  </group>

</launch>
